### DISCUSSION TITLE
Can I Use Drivers, Keys, Themes etc from rEFInd?

### DISCUSSION THREAD

**Discussion by 1457384613gh** [dakanji/RefindPlus#53] [General]
**Describe the Bug:**

What about drivers, keys, themes or so on?
May I copy those from rEFInd?
Additionally, how about the relative path to the folder named keys? 
And in order to boot Chrome OS on Brunch Framework, may I add [brunch.ber](https://github.com/sebanc/brunch/blob/master/brunch.der) into the folder named keys?

**Comment by dakanji** [comment]
Anything from rEFInd can be used in RefindPlus. If you read the [Installation Section](https://github.com/dakanji/RefindPlus#installation) of the ReadMe, you would notice that installing rEFInd and then simply replacing the efi is a way to implement RefindPlus.

On ChromeOS, you can use whatever procedure works with rEFInd.
